.changeset/mean-rats-prove.md

```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/polaris': minor
+---
+
+Fixed issue where DataTable cells would not wrap into multiple lines. `truncate` prop can now be used without `fixedFirstColumns`
```

Comment:
```suggestion
- Fixed `DataTable` cell content not wrapping when the `wrap` prop is set to `true`
- Added support for setting the `DataTable` `truncate` prop without having to set the `fixedFirstColumns` prop
```

Reply:
Chloe, I refrained from doing bullets here because they always end up looking strange in the Changelog. I think having it two sentences looks better.

<img width="938" alt="Screen Shot 2022-08-30 at 10 05 21 AM" src="https://user-images.githubusercontent.com/4960217/187458322-533a080d-34c2-4e99-aadf-13bb2f5212c2.png">

Reply:
Looking at this, I could probably do one sentence and then have bullets underneath. Will do that.